components/tikv_util/src/sys/cgroup.rs

```diff
@@ -189,9 +189,11 @@ fn parse_proc_cgroup_v1(lines: &str) -> HashMap<String, String> {
         let mut iter = line.split(':');
         if let Some(_id) = iter.next() {
             if let Some(systems) = iter.next() {
-                if let Some(path) = iter.next() {
+                // If the path itself contains ":", we need to concat them
```

Comment:
Maybe you should add some comment on the function annotation?

Reply:
Done.